extensions/wikia/UserLogin/css/UserLoginDropdown.globalNavigation.scss

```diff
@@ -54,7 +54,7 @@
 	}
 
 	.WikiaForm {
-		padding: 0 $large-content-padding;
+		padding: 0 $large-content-padding $large-content-padding;
```

Comment:
I just checked this out and it looks like there's some extra padding in the drop down now that wasn't there before
![screen shot 2015-01-14 at 2 26 50 pm](https://cloud.githubusercontent.com/assets/201367/5749069/24bd0e3e-9bfa-11e4-954c-d2f3b2e96fe7.png)
![screen shot 2015-01-14 at 2 32 26 pm](https://cloud.githubusercontent.com/assets/201367/5749075/304930ac-9bfa-11e4-8a48-9c25cee8711c.png)